spec/jobs/rapid_ready_for_decision/form526_base_job_spec.rb

```diff
@@ -116,6 +116,17 @@
             end
           end
         end
+
+        it 'skips forwarding the claim to MAS' do
+          VCR.use_cassette('evss/claims/claims') do
+            Sidekiq::Testing.inline! do
+              expect(MailAutomation::Client).not_to receive(:new)
```

Comment:
To DRY the code, consider moving this line to the prior test and renaming the prior test `'off-ramps to the non-RRD process and does not notify MAS'`, and removing this test.